### DISCUSSION TITLE
anosim() vs adonis2()?

### DISCUSSION THREAD

**Discussion by marwa38** [vegandevs/vegan#586] [Q&A] [answered]
**In case of testing for interactions between two factors which one to use? anosim() or adonis2()?**

I got the same results when I ran adonis2() or anosim() but for interaction both gave different results?
Any inputs which I should use if both functions provide interaction between groups?

my codes 
```
anosim(dist.bray.intes.log, grouping = interaction(meta.ps.prev.intes.log$Group, meta.ps.prev.intes.log$Phase))
Dissimilarity: bray 

ANOSIM statistic R: 0.3738 
      Significance: 0.001 

Permutation: free
Number of permutations: 999

adonis2(dist.bray.intes.log ~ Group * Phase, data = meta.ps.prev.intes.log)

Permutation test for adonis under reduced model
Terms added sequentially (first to last)
Permutation: free
Number of permutations: 999
```

```
adonis2(formula = dist.bray.intes.log ~ Group * Phase, data = meta.ps.prev.intes.log)
             Df SumOfSqs      R2      F Pr(>F)    
Group         1    0.541 0.01658 1.9976  0.020 *  
Phase         2    4.621 0.14154 8.5251  0.001 ***
Group:Phase   2    0.655 0.02006 1.2080  0.138    
Residual     99   26.834 0.82182                  
Total       104   32.652 1.00000  
```

many thanks in advance

**Comment by jarioksa** [comment, marked as answer]
`anosim` gives you only the overall test. It doesn't test separately main effects and interaction, but they are all pooled together into one overall test. The `interaction()` does not give you interactions *after* main effects but effects of all terms together, including main effects. Similar analysis can be found in `adonis2` with argument `by = NULL`  which assesses all terms together. This is not a particular design feature in **vegan** but that is how linear algebra works. If you want to get specifically main effects and interaction, you should use `adonis2` (in general,  it is always better to use `adonis2` than `anosim`).

**Comment by marwa38** [comment]
@jarioksa Dear Jari,
Is it expected to have different results when we do pairwise comparisons using ANSOIM and PERMANVOA (adonis)? I worked on another dataset and got different results between both. Overall, ANOSIM gave more significant results than PERMANVOA.

**Reply by jarioksa** [reply]
First, we don't have pairwise comparisons with `anosim` or `adonis2` in **vegan**, and I won't comment on those. Second, `anosim` is more sensitive to shapes and spreads of point clouds than `adonis2`. Usually we hope to test differences in location (differences in means), but tests can be confounded by differences in spreads  or sizes of clouds ("variances") and shapes ("skewness"), in particular in `anosim`. Less so in `adonis2`, although it also is influenced by shapes and spreads. Therefore we recommend `adonis2` and warn against using `anosim`.

**Reply by marwa38** [reply]
Many Thanks for your comment @jarioksa

**Reply by marwa38** [reply]
@jarioksa 
I split my data beforehand into pairs so that I could run pairwise comparisons using anosim() and adonis2() [vegan package]. Why ANOSIM is there in the first place in vegan if you recommend against using them at all? 
Can't I use both to complement each other? Using adonis2 for the general effect and interactions while using anosim for the pairs of comparisons as it is more sensitive to shapes and spreads of point clouds?

So I would say something like this (if you need something in context to get my point):

- Both PERMANOVA and ANOSIM consistently show significant differences in beta diversity between the pyloric and distal gut regions. This indicates that these two gut regions have distinct microbial compositions, and this difference is detectable by both statistical tests.
- Both PERMANOVA and ANOSIM indicate no significant differences between the MMV and VMV dietary groups concerning the overall microbial community structure. This suggests that, on the whole, the microbial compositions in both dietary groups are similar.
- Despite not finding significant differences in the overall microbial community structure between the MMV and VMV groups, ANOSIM identifies specific pairwise differences within the VMV group.
- ANOSIM shows that the microbial composition between the pyloric and middle gut regions in the VMV group is significantly different. Additionally, it indicates significant differences between the middle and distal gut regions in the VMV group.
- These pairwise differences imply that, although the overall microbial community structure is not significantly different between the MMV and VMV groups, there are specific gut region pairs that exhibit dissimilarities in the VMV group.
- The differences observed between ANOSIM and PERMANOVA demonstrate the importance of understanding the nuances of different statistical methods and their underlying assumptions. While PERMANOVA assesses the multivariate dispersion between groups as a whole, ANOSIM focuses on rank distances between pairs of samples. The pairwise approach of ANOSIM allows for a more detailed examination of specific comparisons.
- The significant pairwise differences detected by ANOSIM in the VMV group indicate that despite not showing a significant difference at the group level (whole microbial composition), there are distinct microbial patterns between certain gut region pairs in the VMV group. This suggests that the VMV dietary intervention has a detectable influence on the microbial community structure within specific gut regions.
- Overall, the combination of PERMANOVA and ANOSIM provides a more comprehensive understanding of the data. While PERMANOVA tells us about the overall differences between groups (MMV vs. VMV), ANOSIM allows us to explore finer distinctions between specific gut region pairs (e.g., pyloric vs. middle and middle vs. distal) within the VMV group.
- These findings highlight the complexity of gut microbial ecology and how different statistical tests can provide complementary insights. The significant pairwise differences found by ANOSIM in the VMV group indicate that despite not showing overall differences between MMV and VMV groups, there are meaningful changes at the level of specific gut regions. These insights can inform further research into the potential regional effects of dietary interventions on the gut microbiome and its impact on host health.
